pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/notification/PillarboxMediaDescriptionAdapter.kt

```diff
@@ -67,7 +69,9 @@ class PillarboxMediaDescriptionAdapter(
                 val imageUri = player.mediaMetadata.artworkUri!!
                 val artworkBitmap = bitmapCache.get(imageUri)
                 if (artworkBitmap == null) {
-                    loadBitmapFromUri(imageUri, callback)
+                    MainScope().launch {
+                        loadBitmapFromUri(imageUri, callback)
+                    }
                     bitmapCache.get(imageUri) // FIXME could return placeholder.
```

Comment:
Now that the call is asynchronous, this could be just `null`, no?
```suggestion
                    null // FIXME could return placeholder.
```